lib/wor/paginate/formatters/panko_formatter.rb

```diff
@@ -0,0 +1,20 @@
+module Wor
+  module Paginate
+    module Formatters
+      class PankoFormatter < Base
+        def serialized_content
+          raise Wor::Paginate::Exceptions::DependencyError unless valid_serializer
+          ActiveRecord::Base.transaction do
+            Panko::ArraySerializer.new(paginated_content, each_serializer: serializer).to_a
+          end
```

Comment:
why is a transaction needed here?

Reply:
🤔 i don't remember why needed this, but I have a slight memory that I used it because it has a problem with ActiveRecord